mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -62,6 +64,26 @@
 eog = ['HL', 'HR', 'Vb']
 
 
+def test_montage():
+    """Test montage."""
+    # read and set up the digitzation
+    original = read_raw_brainvision(vhdr_path, montage=None, eog=eog,
+                                    misc='auto')
+    raw = read_raw_brainvision(vhdr_path, montage=None, eog=eog, misc='auto')
+    assert raw.info['dig'] is None  # No montage inside vhdr_path
+    raw.set_montage(montage)
+    assert isinstance(raw.info['dig'], Digitization)
+    assert raw.info['dig']
+    assert object_diff(raw.info['chs'], original.info['chs'])
```

Comment:
I had to do `original = read_...` because that would not give me a deep copy:

```py
original_chs = raw.info['chs'].copy()
...
assert object_diff(raw.info['chs'], original)
```

Reply:
`info['chs']` is just a list so you can do `original_chs = deepcopy(raw.info['chs'])` or rely on the `info.copy` method that uses `deepcopy` under the hood by doing `original_chs = raw.info.copy()['chs']`. But I'd use the first one because it'll make only the necessary copies

Reply:
or maybe it should be used something like in here:
https://github.com/mne-tools/mne-python/blob/9dee0a7a37aeaf22d4ad9ddaf916f96802514340/mne/channels/tests/test_montage.py#L533